packages/server/src/api/controllers/user.ts

```diff
@@ -173,3 +173,30 @@ export async function getFlags(ctx: BBContext) {
   }
   ctx.body = doc
 }
+
+export async function removeUserFromApp(ctx: BBContext) {
+  const { id: userId, prodAppId } = ctx.params
+
+  const devAppId = dbCore.getDevelopmentAppID(prodAppId)
+  for (let appId of [prodAppId, devAppId]) {
+    if (!(await dbCore.dbExists(appId))) {
+      continue
+    }
+    await context.doInAppContext(appId, async () => {
+      const db = context.getAppDB()
+      const metadataId = generateUserMetadataID(userId)
+      let metadata
+      try {
+        metadata = await db.get(metadataId)
+      } catch (err) {
+        console.warn(`User cannot be found in the app`, { userId, appId })
```

Comment:
I kept is as a warn and return because of the similar functionality on the sync function above. Do you think we should throw a 400 in there as well? Or might we have a different use case?

Reply:
I think for the sync endpoints we decided just not to error, as its nots a specific user action, I think in this case I would just replace the `console.warn` with a `logWarn` instead.